sample/lib/main.dart

```diff
@@ -73,15 +74,18 @@ class HomeScreen extends StatelessWidget {
             FractionallySizedBox(
               widthFactor: 0.5,
               child: Image.asset(
-                Assets.images.nimbleLogo.path,
+                Assets.images.nimbleLogoHorizontal.path,
                 fit: BoxFit.fitWidth,
               ),
             ),
             const SizedBox(height: 24),
             Text(AppLocalizations.of(context)!.hello),
             Text(
               FlutterConfig.get('SECRET'),
-              style: const TextStyle(color: Colors.black, fontSize: 24),
+              style: const TextStyle(
+                color: ColorName.nimblePrimaryBlue,
+                fontSize: 24,
+              ),
```

Comment:
We should not commit & push changes for the sample project. It's automatically managed by the [generate_sample](https://github.com/nimblehq/flutter-templates/blob/develop/.github/workflows/generate_sample.yml) workflow.